tests/cli/test_main_run.py

```diff
@@ -120,6 +120,84 @@ def test_multiline_run_command(tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixtur
         assert not stderr
 
 
+@pytest.mark.parametrize(
+    "args,expected_output",
+    [
+        # no separator: arguments after the executable are passed through to the executable
+        pytest.param(["small", "-v", "-c", "spam"], "-v -c spam", id="no separator"),
```

Comment:
@kenodegard, this was previously expecting `-c spam` in the output after your refactor in the previous commits, but I've updated it to see `-v -c spam`. I think your assumption at that time may have been that conda will consume and remove `-v`, but we're using `nargs=REMAINDER,` here, which means that once the executable token is encountered, all following tokens (even ones that look like options) are part of the executable argv and they get passed through without changes.

So, `conda run small -v -c spam` means that once `conda run` sees `small`, it’s treating that as the start of the positional `executable_call`.

I think if any user asks later on, we should make it unambiguous that anything after `--` is not for `conda run` to see, if it isn't as much already.

Reply:
Aaah this is reviving some dormant braincells. Yes, I see now that the pre-parser will do an initial passthrough but it doesn't consume any arguments (should it??? or did I already try something like that in the past 🤔).

Without `--`, `-vvv` gets interpreted twice (once as debug verbosity level and once as an argument for `small`):

<img width="1085" height="84" alt="Image" src="https://github.com/user-attachments/assets/383a11b9-e339-403b-a09b-b853ccb14654" />

With `--`, `-vvv` is only interpreted once:

<img width="800" height="47" alt="Image" src="https://github.com/user-attachments/assets/dd01344c-c7c6-402f-8b2d-89ffa01b5219" />

I suppose we should add these two test cases and add assertions for stderr.